nautobot/core/views/mixins.py

```diff
@@ -743,10 +743,11 @@ def _process_destroy_form(self, form):
         request = self.request
         obj = self.obj
         queryset = self.get_queryset()
+        msg = ""
```

Comment:
Looks unnecessary here - we only use `msg` inside the `transaction.atomic` block below?